pkg/metrics/metrics.go

```diff
@@ -11,28 +9,36 @@ const (
 
 	sharesSubsystem = "openshift_csi_share"
 
-	mount          = "mount"
-	mountCountName = sharesSubsystem + separator + mount + separator + "total"
+	mount                 = "mount"
+	mountCountName        = sharesSubsystem + separator + mount + separator + "requests_total"
+	mountFailureCountName = sharesSubsystem + separator + mount + separator + "failures_total"
 
 	MetricsPort = 6000
 )
 
 var (
-	mountCounter = createMountCounter()
+	mountCounter, failedMountCounter = createMountCounter()
 )
 
-func createMountCounter() *prometheus.CounterVec {
-	return prometheus.NewCounterVec(prometheus.CounterOpts{
-		Name: mountCountName,
-		Help: "Counts share volume mount attempts by success. " +
-			"'succeeded' label will hold 'true' in case of succeeded mount, and 'false' otherwise.",
-	}, []string{"succeeded"})
+func createMountCounter() (prometheus.Counter, prometheus.Counter) {
```

Comment:
Style nit - pluralize:
```suggestion
func createMountCounters() (prometheus.Counter, prometheus.Counter) {
```